docs/bigquery/snippets.py

```diff
@@ -491,7 +491,9 @@ def do_something(row):
     ]
     assert len(rows) == total == 2
     assert token is None
-    assert rows == expected_rows
+    # Order is not preserved, so compare individually
+    for row in expected_rows:
+        assert row in rows
```

Comment:
You also need a length check?

Reply:
The length check is above.

Reply:
Ah MERP! Since there is only two, it seems a for loop is "overkill"

Reply:
Eh, yeah, still a useful assertion I think.

Reply:
The `for` loop isn't an assertion? I'm saying, just do:

```python
row1 = bigquery.Row(('Wylma Phlyntstone', 29), {'full_name': 0, 'age': 1})
assert row1 in rows
row2 = bigquery.Row(('Phred Phlyntstone', 32), {'full_name': 0, 'age': 1})
assert row2 in rows
```

Reply:
Oh I see, easy enough. Done in e07a8620b.